### PR TITLE
Update to Asp.Net Core 3.0 Preview 4

### LINKED ISSUE #17: Update to .net 3.0 preview 4
Currently the latest version is not compatible with .net core 3.0 preview 3. Any plans to update to preview 4 ?

### LINKED ISSUE #18: Replace licenseUrl MS-Build property
When building the solution with .Net Core 3.0 Preview 4, I get a build error
```
C:\Program Files\dotnet\sdk\3.0.100-preview4-011223\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(199,5): error NU5105: The package version '0.5.0-blazor-0.9.0-preview3-19154-02' uses SemVer 2.0.0 or components of SemVer 1.0.0 that are not supported on legacy clients. Change the package version to a SemVer 1.0.0 string. If the version contains a release label it must start with a letter. This message can be ignored if the package is not intended for older clients. [C:\Users\at\Source\Repos\BlazorSignalR\src\BlazorSignalR\BlazorSignalR.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview4-011223\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(199,5): error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead. [C:\Users\at\Source\Repos\BlazorSignalR\src\BlazorSignalR\BlazorSignalR.csproj]
```   

The error should either be ignored or MS-Build property `licenseUrl` replaced with the `license` MS-Build property.

### PR DESCRIPTION
**Pull request by CatoLeanTruetschel**
This updates the project to (Asp).Net Core 3.0 Preview 4, therefore fixes #17   

Additionally
* ~~this will re-enable logging, as the `BlazorExtensions.Logging` library is now updated too.~~
* this replaces the `licenseUrl` MS-Build propert with the `license` property to fix #18.

### CODE REVIEW

**Comment by CatoLeanTruetschel** [comment]
> this will re-enable logging, as the `BlazorExtensions.Logging` library is now updated too.  

The lib is updated but not packaged yet, so this will not get included.

File: test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs [hunk 1]
```diff
@@ -86,7 +86,7 @@ public void DemoMethodList(DemoData[] data)
 
         private async Task<string> GetJwtToken(string userId)
         {
-            var httpResponse = await this._http.GetAsync($"/generatetoken?user={userId}");
+            var httpResponse = await this._http.GetAsync($"{_http.BaseAddress}generatetoken?user={userId}");
```
Comment (csnewman) [inline test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs]: Is the base address guaranteed to end with a '/'? If not, there's no harm in putting an extra one anyways to just be sure.

Comment (CatoLeanTruetschel) [reply test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs]: I think so. I had there a / before. The server does not route this correctly then. But I can insert a check and add a / conditionally.

File: Directory.Build.props [hunk 2]
```diff
@@ -20,6 +20,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <!-- We use full (Windows PDBs) until cross platform support for source link will get better -->
     <DebugType>full</DebugType>
+     <NoWarn>NU5105</NoWarn>
```
Comment (csnewman) [inline Directory.Build.props:23]: Which warning is this suppressing?

File: test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs [hunk 3]
```diff
@@ -39,8 +39,8 @@ protected override async Task OnInitAsync()
 
             factory.WithUrlBlazor("/chathub", _jsRuntime, options: opt =>
             {
-//                opt.Transports = HttpTransportType.WebSockets;
-//                opt.SkipNegotiation = true;
+                //                opt.Transports = HttpTransportType.WebSockets;
```
Comment (csnewman) [inline test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs]: Formatting

**Review by csnewman — CHANGES_REQUESTED** [review]

File: Directory.Build.props (hunk 2 above)
Comment (CatoLeanTruetschel) [reply Directory.Build.props:23]: This is a warning that you are using a nuget version that is not SemVer 1.0.0 compatible.

```
error NU5105: The package version '0.5.0-blazor-0.9.0-preview3-19154-02' uses SemVer 2.0.0 or components of SemVer 1.0.0 that are not supported on legacy clients. Change the package version to a SemVer 1.0.0 string. If the version contains a release label it must start with a letter. This message can be ignored if the package is not intended for older clients.
```

**Comment by csnewman** [comment]
Good job, thanks for handling this.

I'll get a release out later today.

**Comment by csnewman** [comment]
@AndreasTruetschel ```0.6.0-blazor-3.0.0-preview4-19216-03``` has been released.

**Pull request merged**